### ISSUE TITLE
Suez Canal

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2398]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Table**
https://arctos.database.museum/geography.cfm

**Value**
Suez Canal

**Definition**
The Suez Canal (Arabic: قناة السويس‎ qanāt as-suwēs) is a sea-level waterway in Egypt, connecting the Mediterranean Sea to the Red Sea through the Isthmus of Suez. [Wikipedia](https://en.wikipedia.org/wiki/Suez_Canal)

**Context**
Mollusks collected in the canal.

Should Suez Canal be a drainage or a feature or should it be higher geography at all? Assistance appreciated!

**Comment by dustymc** [comment]
>Should Suez Canal be a drainage or a feature 

Depends on the spatial data (which likely isn't available...). As a feature, I'd expect it to be mostly/entirely water. As a drainage, it would be include all of the dirt "uphill" of the water and tributaries to the canal and such. (Which sets us up to have both, spelled the same way, which won't make it past the unique constraint!)

>should it be higher geography at all

I don't think we have a useful way to answer that, but this one seems useful to me.

**Comment by Jegelewicz** [comment]
Agree and I am leaning towards feature.

**Comment by dustymc** [comment]
>new code table

I think no - and I'd get rid of ctfeature if ya'll would let me,  it's not doing anything useful.

>Waterbody

That part I'm fine with, but it needs a new Issue (and a definition that makes it unambiguously distinct from our other not-quite-feature things, at least if we're still trying to do what I think we should be trying to do).

>Add a field 

Let's not confound things and prevent ourselves from entering the wet parts of national parks and such.

**Comment by Jegelewicz** [comment]
>  it's not doing anything useful.

I disagree. It helps to organize stuff that crosses other higher geography boundaries. A single spelling for "Pacific Remote Islands Marine National Monument" is helpful because it could be written out a million different ways. It isn't perfect, but I think it is better that "write whatever you want here".



>     Waterbody
> 
> That part I'm fine with, but it needs a new Issue (and a definition that makes it unambiguously distinct from our other not-quite-feature things, at least if we're still trying to do what I think we should be trying to do).

I'm not sure I understand. If you don't want a new code table or a way to mark bodies of water in CTFEATURE, where would this go and what would it do?

**Comment by dustymc** [comment]
We only let people who can spell have access to geography, right?!? (But yea, it's useful for that. Or I don't find many misspellings for the other components that aren't controlled. Whatever, I can go either way.)

>where would this go 

A new "column" in geog_auth_rec - which would let you say "Whatever National Park (feature), Some Drainage (drainage), Some Lake (waterbody)." And maybe do something useful for bays-and-such, but...

>what would it do

I don't know, I still think we need to back WAY up, decide on what exactly we're trying to do with geography, and build on that.

**Comment by Jegelewicz** [comment]
Also thinking about [Rio Saladillo](https://en.wikipedia.org/wiki/Saladillo_Stream)

Again, the rivers, lakes, etc seem to be calling out for waterbody.....

**Comment by Jegelewicz** [comment]
I think I am going to add these as features for now. The topic of geography should be a part of our post PG transition discussion.

**Comment by Jegelewicz** [comment]
BUT it would be nice if all higher geography included coordinates and error which could be used if Specific Locality had none....

**Comment by tucotuco** [comment]
Isn't that redundant. A georeference describes the whole location, no
matter what location fields are populated, right?

**Comment by Jegelewicz** [comment]
So much material has no coordinates. Coordinates assigned to higher geography (which we don't need to create because others already have) would at least provide a rough georeference.

**Comment by Jegelewicz** [comment]
Check this out! Searched "Patagonia" [here](http://geonames.nga.mil/namesviewer/). and get this:

![image](https://user-images.githubusercontent.com/5725767/70254981-37860880-1743-11ea-94cc-4cbfca7b2801.png)

It seems like this site has potential usefulness?

**Comment by tucotuco** [comment]
That's fine, but georeferences aren't for geography in the model (unless
something changed drastically). The georeference would be for a "locality
not recorded" joined to your geography. The georef is for that whole combo,
not just for the geography.

**Comment by Jegelewicz** [comment]
> georeferences aren't for geography in the model

That is correct. However, I will bet a lot of money that stuff that isn't georeferenced now at the locality level never will be (unless there is some push by funding agencies to get it done). If stuff gets mapped to a point with a large error (a county), that's better than never being mapped at all, I think. I am actually suggesting a change to the model, at least for presentation and search purposes. If locality has no coordinates, then use coordinates of the higher geography and flag it as such.

**Comment by dustymc** [comment]
>all higher geography included coordinates and error which could be used if Specific Locality had none.

If you mean point-radius I believe it's technically trivial to derive them (from geography WKT), but they might also include a lot of unintended country.

>Coordinates assigned to higher geography (which we don't need to create because others already have) would at least provide a rough georeference.

I have a rough georeference, I think less-rough than that, for ~everything. I could expose it through more than the edit locality form and some search tools - just a social issue.

**Comment by dustymc** [comment]
> georeferences aren't for geography in the model 

We currently have three place-things

* WKT attached to geography spatially defines geography
* point-radius attached to localities circumscribes events
* WTK attached to locality does - uhhh, something, probably.... (We got a bunch of those data from fishnet-or-something, I duct taped it to the side of locality, it needs more discussion.)

**Comment by tucotuco** [comment]
I'm sorry, but that seems like a mess. Why not WKT and P-R as spatial data
for every locality (only). If the locality has only geography, great,
that's what you have attached to geography now.

I don't understand the the argument about funding. It is exactly the same
effort to georeference a locality with only higher geography as it is to
georeference a geography.

**Comment by campmlc** [comment]
I doubt we will ever have enough funding to hire the necessary people to
georeference all localities in all collections - at least not in my
lifetime. I'd love to be proved wrong on this.
I guess I don't understand the current model - it seems to make sense to
use the coordinates of what we do know (e.g. county) if we have no
coordinates or information for specific locality. Right now a huge number
of our specimen records are not mappable because of the lack of coordinates
at the locality level, but we do know higher geography.

**Comment by tucotuco** [comment]
Right. All I am saying is to put those known geography shapes on the
locality table - specifically on the locality for that geography that has
no further information.

**Comment by dustymc** [comment]
>Why not

From my perspective, geography WKT is useful for error detection.

http://arctos.database.museum/Locality.cfm?Action=editGeog&geog_auth_rec_id=1042 (North America, United States, Nebraska) looks like this:

<img width="461" alt="Screen Shot 2019-12-05 at 9 17 16 AM" src="https://user-images.githubusercontent.com/5720791/70257885-0eb04400-1740-11ea-93ef-ab000fd925be.png">

Clicky..

<img width="545" alt="Screen Shot 2019-12-05 at 9 17 46 AM" src="https://user-images.githubusercontent.com/5720791/70257919-1ff95080-1740-11ea-8915-ac8e50d26b90.png">

and...

<img width="1335" alt="Screen Shot 2019-12-05 at 9 18 16 AM" src="https://user-images.githubusercontent.com/5720791/70257954-31daf380-1740-11ea-965d-a7d638aca694.png">

The specimen detail map looks like....

<img width="975" alt="Screen Shot 2019-12-05 at 9 19 23 AM" src="https://user-images.githubusercontent.com/5720791/70258047-5d5dde00-1740-11ea-8187-9a9fd3aa8b99.png">

(And there's an issue to have it auto-submit Annotations somewhere.)

Geography is an authority - the WKT just makes it a useful one.

Specimen-events (including locality/spatial data) are assertions.

The spatial intersection of those two things is informative.

WHY that specimen maps off yonder is something I leave to Curators, and why I'd be generally resistant to the idea (implicit or otherwise) that geography is somehow "correct." Some of them are probably funky georeferences, some of them are mis-transcriptions of geography data, most are probably a bit of both, and resolving that is likely to involve digging through specimen data.

>funding to hire the necessary people to georeference all localities in all collections

It doesn't have to be a people-problem, I already georeference most everything because the data are useful for my purposes. They're there if you want them....

(And there's very likely something interesting about the ones I can't georeference too.)

```
UAM@ARCTOS> select count(*) from locality;

  COUNT(*)
----------
    594987

1 row selected.

Elapsed: 00:00:00.44
UAM@ARCTOS> select count(*) from locality where dec_lat is null;

  COUNT(*)
----------
    153718

1 row selected.

Elapsed: 00:00:00.41
UAM@ARCTOS> select count(*) from locality where s$dec_lat is null;

  COUNT(*)
----------
     15786

1 row selected.
```

**Comment by tucotuco** [comment]
Good arguments.

**Comment by Jegelewicz** [comment]
> I don't understand the the argument about funding. It is exactly the same
> effort to georeference a locality with only higher geography as it is to
> georeference a geography.

I think we are just talking around each other. I am suggesting exactly that we use the wkt info that is already available NOT that WE georeference higher geography.

**Comment by dustymc** [comment]
>but we do know higher geography.

I'll be more explicit: I don't believe you. Zoom in on the Nebraska map and it starts to look like...

<img width="314" alt="Screen Shot 2019-12-05 at 9 34 12 AM" src="https://user-images.githubusercontent.com/5720791/70259142-749dcb00-1742-11ea-8baf-5048acc752f3.png">

How sure are you that the specimen is not in fact from SD? If the specimen wasn't georeferenced and we assumed the geography was right, there'd be very little way of knowing there was some ambiguity - it'd just map to central NE with a giant circle and nobody would ever question anything.

I'm absolutely not suggesting we don't push some data downstream, just that we need to somehow be very explicit in where it came from.

And for most things I have better data than that from geography - it came from services that also consider things like specloc.

**Comment by Jegelewicz** [comment]
> How sure are you that the specimen is not in fact from SD? If the specimen wasn't georeferenced and we assumed the geography was right, there'd be very little way of knowing there was some ambiguity - it'd just map to central NE with a giant circle and nobody would ever question anything.

To me, this a perfect argument for never mapping anything that hasn't been georeferenced. I'd be on board with that and maybe it would force some funding agencies to offer up grants for getting it done properly.

> I'm absolutely not suggesting we don't push some data downstream, just that we need to somehow be very explicit in where it came from.

That's why I said it should be flagged - not that it matters, people will ignore what they don't want to see or don't pay enough attention to see.

> And for most things I have better data than that from geography - it came from services that also consider things like specloc.

Then we need to figure out how to leverage that and get some Arctos-wide funding to use it to georeference stuff.

**Comment by dustymc** [comment]
> leverage that 

There's a tiny little lever in edit locality - you can push the service data to the actual locality with a click, and it'll report the distance between asserted and derived point (and I think there's something about elevation). Doing more is likely trivial from a technical standpoint.

There have been a bunch of projects to georeference various things - it might be possible to fund another one.

My georeferences are a fairly primitive afterthought, and they're still pretty good. I'd prefer to see what a well-funded service can do before we start paying people to click buttons. I think people would still be a critical component, but they should be dealing with the "right on the border" type things and not the "Whatever County, Texas" stuff that's trivial for automation.

**Comment by Jegelewicz** [comment]
I don't remember who needed this - closing as duplicate of #2876
